chunky/src/java/se/llbit/chunky/ui/render/WaterTab.java

```diff
@@ -92,17 +83,25 @@ public WaterTab() throws IOException {
     waterColor.colorProperty().removeListener(waterColorListener);
     waterColor.setColor(ColorUtil.toFx(scene.getWaterColor()));
     waterColor.colorProperty().addListener(waterColorListener);
+
+    waterPlaneEnabled.setSelected(scene.isWaterPlaneEnabled());
+    waterPlaneHeight.setRange(scene.yClipMin, scene.yClipMax);
+    waterPlaneHeight.set(scene.getWaterPlaneHeight(true));
```

Comment:
Oh, _that's_ what the method is there for. It's weird that the getter doesn't return what the setter sets… Maybe `getEffectiveWaterPlaneHeight()` for the one that takes the offset into account – what do you think? :thinking: